### PR TITLE
root: dockerize app

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -27,6 +27,35 @@ services:
...
+      dockerfile: ./Dockerfile
+    image: tusharchy/nest-and-postgres-application:latest
+    environment:
+      - DB_TYPE=${DB_TYPE}
+      - DB_HOST=${DB_HOST}
+      - DB_PORT=${DB_PORT}
+      - DB_USERNAME=${DB_USERNAME}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_NAME=${DB_NAME}
+      - POSTGRES_SYNC=false
```
Comment (mmiranda-stackit) [inline docker-compose.yml]: What is POSGRES_SYNC and what is used for?

File: docker-compose.yml [hunk 2]
```diff
@@ -27,6 +27,35 @@ services:
...
+      - JWT_SECRET=${JWT_SECRET}
+      - PGADMIN_DEFAULT_EMAIL=${PGADMIN_DEFAULT_EMAIL}
```
Comment (mmiranda-stackit) [inline docker-compose.yml]: The nest app won't need the PG ADMIN keys I believe.

File: docker-compose.yml [hunk 3]
```diff
@@ -6,13 +6,13 @@ services:
     ports:
       - '5432:5432'
     environment:
-      - DATABASE_HOST=127.0.0.1
+      - DATABASE_HOST=${DB_HOST}
```
Comment (mmiranda-stackit) [inline docker-compose.yml:9]: nice

File: docker-compose.yml (hunk 2 above)
Comment (AlefrankM) [reply docker-compose.yml]: You believe well

**Review by mmiranda-stackit — APPROVED** [review]
lgtm

**Pull request merged**